### DISCUSSION TITLE
Elastic 7.9.3 has been released

### DISCUSSION THREAD

**Discussion by TOoSmOotH** [Security-Onion-Solutions/securityonion#1624] [Announcements]
Elastic 7.9.3 has been released and Elastic is calling it a security update. We will be releasing 2.3.2 in the coming days to address this issue. We still recommend upgrading to 2.3.1 in the interim. 

https://www.elastic.co/blog/elastic-stack-7-9-3-released

**Comment by dougburks** [comment]
Also see https://github.com/Security-Onion-Solutions/securityonion/issues/1618

**Comment by TOoSmOotH** [comment]
2.3.2 has been released!

https://blog.securityonion.net/2020/10/security-onion-232-now-available.html
